CMakeLists.txt

```diff
@@ -467,6 +467,12 @@ ENDIF()
 ADD_EXECUTABLE(${TARGET} ${TARGET_SRC})
 DEAL_II_SETUP_TARGET(${TARGET})
 
+IF (ASPECT_PRECOMPILE_HEADERS)
+  IF(CMAKE_VERSION VERSION_GREATER_EQUAL 3.16)
```

Comment:
I am not sure I understand. Wouldn't you run into the FATAL_ERROR 6 lines above?